imports/plugins/core/accounts/client/containers/mainDropdown.js

```diff
@@ -1,14 +1,13 @@
 import { compose, withProps } from "recompose";
-import { registerComponent, composeWithTracker, withCurrentAccount } from "@reactioncommerce/reaction-components";
 import { Meteor } from "meteor/meteor";
 import { Accounts } from "meteor/accounts-base";
 import { Roles } from "meteor/alanning:roles";
 import { Session } from "meteor/session";
-import { Reaction, Logger } from "/client/api";
-import { i18nextDep, i18next } from  "/client/api";
+import { registerComponent, composeWithTracker, withCurrentAccount } from "@reactioncommerce/reaction-components";
```

Comment:
We probably need to figure out whether we are treating this as a local import or as an npm package as far as import order. I would support importing it as an NPM package since hopefully that's where it will end up soon. We are doing both ways now. Clearly not a huge deal but since we are making everything consistent.

Reply:
This is something I've been considering lately as well. I've ordered the imports that I've changed from `@reactioncommerce/reaction-components` right after any `meteor` imports, but I haven't found a way to use the `import/order` eslint plugin to enforce that yet.

@machikoyasuda we should write the import order up in the docs as well now that it's defined a little better.